ansible_rulebook/cli.py

```diff
@@ -97,13 +97,30 @@ def get_parser() -> argparse.ArgumentParser:
         "--websocket-address",
         "--websocket-url",
         help="Connect the event log to a websocket",
+        default=os.environ.get("EDA_WEBSOCKET_ADDRESS", ""),
     )
     parser.add_argument(
         "--websocket-ssl-verify",
         help="How to verify SSL when connecting to the "
         "websocket: (yes|true) | (no|false) | <path to a CA bundle>, "
         "default to yes for wss connection.",
-        default="yes",
+        default=os.environ.get("EDA_WEBSOCKET_SSL_VERIFY", "yes"),
+    )
+    parser.add_argument(
+        "--websocket-access-token",
+        help="Token used to autheticate the websocket connection.",
+        default=os.environ.get("EDA_WEBSOCKET_ACCESS_TOKEN", ""),
+    )
+    parser.add_argument(
+        "--websocket-refresh-token",
+        help="Token used to renew a websocket access token.",
+        default=os.environ.get("EDA_WEBSOCKET_REFRESH_TOKEN", ""),
+    )
+    parser.add_argument(
+        "--websocket-token-address",
+        "--websocket-token-url",
+        help="Url to renew websocket access token.",
+        default=os.environ.get("EDA_WEBSOCKET_TOKEN_ADDRESS", ""),
```

Comment:
@bzwei should the be EDA_WEBSOCKET_TOKEN_URL, if it is using https do we need a ssl_verify option for that address?

Reply:
Here wss and https share the same ssl_verify option. It can't be the case that one requires ssl verification but the other does not, can it?

Reply:
@bzwei Its possible that the resource server and the auth server are different.